src/test/deserialization_tests.cpp

```diff
@@ -142,7 +139,10 @@ class GRPCPredictRequestNegative : public GRPCPredictRequest {};
 TEST_P(GRPCPredictRequestNegative, ShouldReturnDeserializationErrorForPrecision) {
     Precision testedPrecision = GetParam();
     tensorMap[tensorName]->setPrecision(testedPrecision);
-    InferenceEngine::InferRequest inferRequest;
+    InferenceEngine::Core engine;
+    InferenceEngine::CNNNetwork network = engine.ReadNetwork(std::filesystem::current_path().u8string() + "/src/test/dummy/1/dummy.xml");
+    InferenceEngine::ExecutableNetwork execNetwork = engine.LoadNetwork(network, "CPU");
+    InferenceEngine::InferRequest inferRequest = execNetwork.CreateInferRequest();
```

Comment:
Why can't we just use ` InferenceEngine::InferRequest inferRequest;` anymore? Precision check happens before `InputSink::give` so there should be no problem using inferRequest this way. Is the default constructor removed? Are there any other constructors now? Or the only way is using CreateInferRequest?